usecase/usecase-webapi-xwalk-tests/samples/IAPGooglePlay/REAME.md

```diff
@@ -0,0 +1,11 @@
+# In-App Purchase Example for Google Play
+## Introduction
+This example is for testing the in-app purchase API for Google Play.
+## Pre-condition
+* Get the latest ipa.zip from [Crosswalk android extension iap example](https://github.com/crosswalk-project/crosswalk-android-extensions/releases/), unzip it and copy the unzipped files to IAPGooglePlay/iap directory.
+* Make sure Google Play and Google Wallet have been installed in the Android system.
```

Comment:
Only Google Play is required, and we should also reminder tester to connect to a network which is able to access Google Play.